internal/host/plugin/repository_host_catalog.go

```diff
@@ -338,8 +614,10 @@ func toPluginCatalog(ctx context.Context, in *HostCatalog) (*pb.HostCatalog, err
 		return nil, errors.New(ctx, errors.InvalidParameter, op, "nil storage plugin")
 	}
 	hc := &pb.HostCatalog{
-		Id:      in.GetPublicId(),
-		ScopeId: in.GetScopeId(),
+		Id:          in.GetPublicId(),
+		ScopeId:     in.GetScopeId(),
+		Name:        wrapperspb.String(in.GetName()),
+		Description: wrapperspb.String(in.GetDescription()),
```

Comment:
Name and Description should only be set with a wrapperspb.String if in.GetName and in.GetDescription are not empty strings.

Reply:
Updated!